07-statistical-testing.Rmd

```diff
@@ -1 +1,570 @@
+
+- $H_0: \mu_1 = \mu_2$ where $\mu_i$ is the is the mean outcome for group $i$
+- $H_A: \mu_1 \neq \mu_2$
+
+Two sample t-tests can also be *paired* or *unpaired*. If the data come from two different populations (e.g., males vs females), then the t-test run will be an *unpaired* or *independent samples* t-test. *Paired* t-tests occur when the data come from the same population. This is commonly seen with data taken from the same population in two different time periods (e.g., before and after an intervention). 
+
+The difference between using t-tests with non-survey data and with survey data is based on the underlying variance estimation difference. Chapter \@ref(c05) provides the detailed overview of the math behind the mean and sampling error calculations for various sample designs. The functions in the {survey} package will account for these nuances provided the design object is correctly defined.  
+
+### Syntax {#stattest-ttest-syntax}
+
+When we do not have survey data, we may be able to use the `t.test()` function. This function does not allow for weights or the variance structure to be accounted for with survey data. Therefore, when using survey data, we need to use the `svyttest()` function. Many of the arguments are the same between the two functions, but there are a two key differences:
+
+- We need to use the survey design object, instead of data
+- We can only use a formula and not separate x and y data
+
+Here is the syntax for the `svyttest()` function:
+
+``` {r}
+#| label: stattest-ttest-syntax
+#| eval: FALSE
+svyttest(formula, 
+         design, 
+         na.rm=FALSE,
+         level=0.95,
+         ...)
+```
+
+Notice that the first argument here is the `formula` and not the `design`. This means that we must use the dot `(.)` if we pipe in the survey design object (as described at the beginning of this chapter). 
+
+The `formula` argument can take on a couple of different forms depending on what it is that we are measuring.  Here are a few common scenarios:
+
+1. **One-sided t-test:** 
+    a. **Comparison to 0:** $var \sim 0$, where $var$ is the measure of interest and $0$ is the value we are comparing it to. For example, we could test if the proportion of the population that has blue eyes is different from $0$.
+    b. **Comparison to a different value:** $I(var - value) \sim 0$, where $var$ is the measure of interest and $value$ is what we are comparing to. We need to use the `I()` function, to tell the program to calculate the difference between the variable and the comparison value prior to testing. For example, we could test if the proportion of the population that has blue eyes is different from $25%$.
+2. **Two-sided t-test:**
+    a. **Unpaired:**
+        - **2 level grouping variable:** $var \sim groupVar$, where $var$ is the measure of interest and $groupVar$ is a variable with two categories.  For example, we could test if the perfect of the popualtion that has blue eyes is different for men and women. 
```

Comment:
Note here or elsewhere: The group variable must be a factor or character with two levels, or be coded 0/1 or 1/2